src/Engine/Manager/PhysicsManager.cpp

```diff
@@ -55,5 +80,26 @@ void PhysicsManager::Update(World &world, float deltaTime) {
         entity->rotation += physicsComp->angularVelocity * 360.f * deltaTime;
     }
 
-    this->Simulate(deltaTime);
+    dynamicsWorld->stepSimulation(deltaTime, 10);
+
+    for (auto trigger : triggers) {
+        trigger->Process(*dynamicsWorld);
+    }
+}
+
+Physics::RigidBody* PhysicsManager::MakeRigidBody(Physics::Shape* shape, float mass) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'MakeRigidBody' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)